notes/lecture26.tex

```diff
@@ -0,0 +1,224 @@
+\section{Primal Dual IPM}
+Previously, we discussed Barrier Methods and the so-called short-step method for solving constrained LPs, and proved that convergence is guaranteed (albeit slow). Herein, we study a Primal-Dual Interior Point method (the so-called "long-step"  path following algorithm), which similarly seeks to approximate points on the central path. Unlike the short-step method, the long-step method considers iterates of Primal-Dual variables and seeks a more aggressive step size so long as it lies within a neighborhood of the central path.
+
+
+
+\subsection{Deriving the dual problem}
+
+Let $x \in \R^n$ be the decision variable, and $A \in \R^{m \times n},b \in \R^m$ and $c \in \R^n$
+\begin{eqnarray}
+\min c^\top x \quad \text{s.t.}\quad Ax = b, x \ge 0 
+\end{eqnarray}
+$\ge$ is pointwise.
+
+Observe that we can always write the constraint as 
+\begin{eqnarray}
+&& \min_{x \ge 0 } c^\top x + \max_{z} z^{\top}(b-Ax) \\
+&=& \min_{x \ge 0} \max_{z} c^\top x + \max_{z} z^{\top}(b-Ax) \\
+&\ge& \max_{z} z^{\top}b - \min_{x \ge 0} (c - A^{\top}z)^{\top}x\\
+&=& \max_{z} z^{\top}b  - \infty \cdot \Ind(A^{\top}z > c)\\
+\end{eqnarray}
+Hence, the dual problem is as follows
+\begin{eqnarray*}
+\max b^\top z \quad \text{s.t.}\quad A^{\top} z \le c
+\end{eqnarray*}
+This is equivalent to 
+\begin{eqnarray}
+\max b^\top z \quad \text{s.t.}\quad A^{\top} z + s = c, \quad s \ge 0
+\end{eqnarray}
+where we introduce the slack variable $s \in \R^n$. If $(x,z,s)$ are only \emph{feasible}, then 
+\begin{eqnarray}
+Ax = b \quad A^{\top} z + s = c \quad x,s \ge 0
+\end{eqnarray}
+Moreover, we can compute that for feasible $(x,z,s)$,
+\begin{eqnarray}
```

Comment:
As a rule of thumb, only use numbered equations if you're going to refer to them later.